Resources/Prototypes/DeltaV/Actions/types.yml

```diff
@@ -34,3 +34,13 @@
       sprite: Structures/Furniture/Tables/generic.rsi
       state: full
     event: !type:ToggleCrawlingStateEvent
+
+- type: entity
+  id: ActionPrecognition
+  name: action-name-precognition
```

Comment:
dont do this, it can be localized with entity localization
just put the actual strings in the prototype